### DISCUSSION TITLE
Is Github Self-Hosted Runners Free?

### DISCUSSION THREAD

**Discussion by mayank** [community/community#30883] [Actions] [answered]
There is no mention of pricing about Github self-hosted runners. What is the pricing if I run a self-hosted MacOS Github Runner for 4-5 hours a day.

**Comment by vitalykarasik** [comment, marked as answer]
Self-hosted runner - it's **your** servers, so obviously you don't pay for them to Github.  In my projects, I run them on GCP, so it's not free for my customer, but if you already have server, you can launch self-hosted runner for free.

**Reply by mayank** [reply]
Thanks! for any operating systems it's free right, even for mac?

**Reply by vitalykarasik** [reply]
Right!
